src/main/resources/script.sh

```diff
@@ -444,14 +444,15 @@ function downloadLFN {
 
   local totalTimeout=$((timeout + srmTimeout + sendReceiveTimeout))
 
-  local LINE="time -p dirac-dms-get-file -d -o /Resources/StorageElements/GFAL_TIMEOUT=${totalTimeout} ${LFN}"
+  local LINE="dirac-dms-get-file -d -o /Resources/StorageElements/GFAL_TIMEOUT=${totalTimeout} ${LFN}"
   info "$LINE"
+  local startDownload=$(date +%s)
   (${LINE}) &> get-file.log
 
   if [ $? = 0 ]; then
+    local duration=$(($(date +%s) - startDownload))
```

Comment:
i suggest to also print the duration in the error log when it fails.

Reply:
Actually the info should be accessible from the `info` and `error` logging method before and after the command.